csm/conf/upgrade.py

```diff
@@ -27,6 +29,90 @@ def __init__(self):
         super(Upgrade, self).__init__()
 
     async def execute(self, command):
-        Log.info("Perform upgrade for csm_setup")
-        # TODO: Implement upgrade logic
+        Log.info("Performing upgrade and loading config files")
+        try:
+            Conf.load(const.CONSUMER_INDEX, command.options.get(const.CONFIG_URL))
+            self.load_csm_config_indices()
+            self.load_default_config()
+        except KvError as e:
+            Log.error(f"Configuration Loading Failed {e}")
+            raise CsmSetupError("Could Not Load Url Provided in Kv Store.")
```

Comment:
Could Not Load Url Provided in Kv Store -> Unable to load configurations

Reply:
Resolved